torch/quantization/ns/utils.py

```diff
@@ -227,3 +227,41 @@ def get_target_type_str(node: Node, gm: GraphModule) -> str:
         target_mod = getattr_from_fqn(gm, node.target)
         target_type = str(type(target_mod))
     return target_type
+
+def rekey_logger_info_on_node_name_of_model(
+    results: NSResultsType,
+    model_name: str,
+) -> NSResultsType:
+    """
+    Rekeys the layer name of a results dictionary to use node names
+    from `model_name`.
+
+    For example, transforms
+
+        {'base_op_1_0': {'node_output': {'model_a':
+          [{'ref_node_name': 'linear1', ...}]}}}
+
+    into
+
+        {'linear1': {'node_output': {'model_a':
+          [{'ref_node_name': 'linear1', ...}]}}}
+
+    Note: we cannot use these node names directly because they are not
+    guaranteed to be consistent across models. This is why we extract
```

Comment:
Just wonder in which cases the node names are not consistent across models? Could you add an example if possible?

Reply:
As far as I know there are no guarantees provided by FX on what node names are generated when tracing through the model.  They are the same today, but that could break at any time in the future, and it's not something we can depend on.